script/deploy/deploySingleContract.sh

```
@@ -230,7 +236,75 @@ deploySingleContract() {
+        #   --gas-price "$GAS_PRICE" \
+        #   --priority-gas-price 0 \
+        #   --create "$DEPLOY_CODE" \
+        #   --json)
+        # echo "TX_OUTPUT: $TX_OUTPUT"
+        ###########################################################
+
+        # DRY_RUN=$(DEPLOYSALT=$DEPLOYSALT CREATE3_FACTORY_ADDRESS=$CREATE3_FACTORY_ADDRESS NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX DEFAULT_DIAMOND_ADDRESS_DEPLOYSALT=$DEFAULT_DIAMOND_ADDRESS_DEPLOYSALT DEPLOY_TO_DEFAULT_DIAMOND_ADDRESS=$DEPLOY_TO_DEFAULT_DIAMOND_ADDRESS PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT") DIAMOND_TYPE=$DIAMOND_TYPE forge script "$FULL_SCRIPT_PATH" -f $NETWORK -vvvvv --json -- --max-fee-per-gas "$MAX_FEE" --dry-run)
+        # echo "DRY_RUN: $DRY_RUN"
+        # echo "DRY_RUN done: $DRY_RUN"
+        ###### THIS DID NOT WORK:
+        RAW_RETURN_DATA=$(DEPLOYSALT=$DEPLOYSALT CREATE3_FACTORY_ADDRESS=$CREATE3_FACTORY_ADDRESS NETWORK=$NETWORK FILE_SUFFIX=$FILE_SUFFIX DEFAULT_DIAMOND_ADDRESS_DEPLOYSALT=$DEFAULT_DIAMOND_ADDRESS_DEPLOYSALT DEPLOY_TO_DEFAULT_DIAMOND_ADDRESS=$DEPLOY_TO_DEFAULT_DIAMOND_ADDRESS PRIVATE_KEY=$(getPrivateKey "$NETWORK" "$ENVIRONMENT") DIAMOND_TYPE=$DIAMOND_TYPE forge script "$FULL_SCRIPT_PATH" -f $NETWORK -vvvvv --json --broadcast --skip-simulation -- --max-fee-per-gas "$MAX_FEE" --max-priority-fee-per-gas 0 --gas-limit "$GAS_LIMIT")
```

Shellcheck (0.10.0)

[warning] This assignment is only seen by the forked process.

(SC2097)